modules/web/src/common/dialogs/deleteresource/template.html

```diff
@@ -22,14 +22,24 @@
     <span *ngIf="data.objectMeta.namespace"> &nbsp;in namespace <i>{{ data.objectMeta.namespace }}</i> </span>
     <span>?</span>
   </ng-container>
+  <ng-container>
+    <div class="kd-equivalent-block">
+      <mat-checkbox color="primary"
+                    [(ngModel)]="isDeleteNowChecked"
+                    name="isDeleteNowChecked"
+                    matInput
+                    i18n>Delete now (sets delete grace period to 1 second instead of the default, which is usually 30 seconds)</mat-checkbox>
```

Comment:
I'd add the description to the info icon with tooltip.

Reply:
Happy new year @floreks !

I am not sure it looks better:

![image](https://user-images.githubusercontent.com/297498/211806748-537f0713-f8f0-4aa3-ba78-69f6f4984504.png)


![image](https://user-images.githubusercontent.com/297498/211806802-0b8c54f8-2def-4891-b0b4-b2cc81d3222c.png)

Also, as I mentioned, I did not put both `force` and `now` because they are basically the same, except that `force` is more a `kubectl` thing that a `kubernetes` thing.

Reply:
I think this is fine:

![image](https://user-images.githubusercontent.com/297498/211807358-d7596ea0-415b-4341-a866-75d8fc36f6b5.png)

Reply:
Or how about this ?
![image](https://user-images.githubusercontent.com/297498/211808671-2708f004-0c9f-4025-aa11-c19d6d22a5dd.png)

we could even add the `force`, although I don't personally see any advantage.

Reply:
or

![image](https://user-images.githubusercontent.com/297498/211808955-23c8adf4-a81e-47c5-830f-0a792eb01205.png)

Reply:
@marcosdiez if I did not clicks neither Delete nor Cancel but escape the dialog box, the delete request still send to the backend.